gusto/timestepping/split_timestepper.py

```diff
@@ -153,10 +154,68 @@ def setup_scheme(self):
         if self.io.output.log_courant:
             self.scheme.courant_max = self.io.courant_max
 
+    def setup_prescribed_expr(self, expr_func):
+        """
+        Sets up the prescribed transporting velocity, through a python function
+        which has time as an argument, and returns a `ufl.Expr`. This allows the
+        velocity to be updated with time.
+
+        Args:
+            expr_func (func): a python function with a single argument that
+                represents the model time, and returns a `ufl.Expr`.
+        """
+
+        self.velocity_projection = Projector(
+            expr_func(self.t), self.fields('u')
+        )
+
+        self.is_velocity_setup = True
+
+    def setup_prescribed_apply(self, apply_method):
+        """
+        Sets up the prescribed transporting velocity, through a python function
+        which has time as an argument. This function will perform the evaluation
+        of the transporting velocity.
+
+        Args:
+            expr_func (func): a python function with a single argument that
+                represents the model time, and performs the evaluation of the
+                transporting velocity.
+        """
+
```

Comment:
Do we need to check that the velocity has not already been setup (e.g. by using the other method)?